test/python/quantum_info/operators/test_operator.py

```diff
@@ -681,6 +681,18 @@ def test_reverse_qargs(self):
         state2 = Operator(circ2)
         self.assertEqual(state1.reverse_qargs(), state2)
 
+    def test_drawings(self):
+        """Test draw method"""
+        qc1 = QFT(5)
+        op = Operator.from_circuit(qc1)
+        with self.subTest(msg="str(operator)"):
+            str(op)
+        for drawtype in ["repr", "text", "latex_source"]:
+            with self.subTest(msg=f"draw('{drawtype}')"):
+                op.draw(drawtype)
+        with self.subTest(msg=" draw('latex')"):
+            op.draw("latex")
```

Comment:
I see that this test is the same as for `Statevector` and `DensityMatrix`, so it's probably best to leave it as you've written it (i.e. just assert that no errors are raised), though if there were something sensible we could sensibly assert about the output, that wouldn't hurt.